wiki/Community/Discord_servers/en.md

```diff
@@ -1,9 +1,105 @@
+
+| Name | Owner |
+| :-- | :-- |
+| [osu! Tournament Hub](https://discord.gg/bvhajDC) | [Sikey](https://osu.ppy.sh/users/343057) |
+| [osu!mania Tourney Central](https://discord.gg/WnMcrUnGV5) | [-mint-](https://osu.ppy.sh/users/8976576) |
+
+## Project Loved
+
+[Project Loved](/wiki/Community/Project_Loved) organises its workflows in different servers, divided by the [game mode](/wiki/Game_mode) which they specialise in. These servers aim to facilitate the discussion of prospective picks for each mode.
+
+| Name | Owner |
+| :-- | :-- |
+| [osu! Project Loved](https://discord.gg/gn58Uk5sTE) | [Librarian](https://osu.ppy.sh/users/10083084) |
+| [Project Loved: Taiko](https://discord.gg/GhfjtZ6) | [Hivie](https://osu.ppy.sh/users/14102976) |
+| [osu!catch Project Loved](https://discord.gg/phgtyS4UCh) | [Wesley](https://osu.ppy.sh/users/2407265) |
+| [osu!mania Loved Community](https://discord.gg/Ededv7m) | [Paturages](https://osu.ppy.sh/users/1375479) |
+
+## Skinning
+
+These servers aim to provide safe places for skinners, no matter if they are design experts or brand-new to skinning. Users can expect to find multiple [skinning](/wiki/Skinning) resources, as well as receiving help from other users interested in the matter.
+
+| Name | Owner | Description |
+| :-- | :-- | :-- |
+| [skinship](https://discord.skinship.xyz/) | [RockRoller](https://osu.ppy.sh/users/8388854) | **skinship** is a place to discuss anything involving skinning, while also hosting projects such as [Skinning Contests](/wiki/Contests/Skinning_Contest) and the [Completed Skins Compendium](https://compendium.skinship.xyz/). Users can manage their skinning affairs by reaching out the [skinship bot account](https://osu.ppy.sh/users/32231941). |
+
+## Storyboarding
+
+These servers aim to provide users with resources to help them in their [storyboarding](/wiki/Storyboard) affairs. They are good starting points for beginners who will have an easier time easing themselves in the world of storyboarding while getting help from experienced users.
+
+| Name | Owner | Description |
+| :-- | :-- | :-- |
+| [osu! storyboarder banquet](https://discord.gg/B8NX7YW) | [Sidetail](https://osu.ppy.sh/users/2036217) | The **osu! storyboarder banquet** serves as a place all about storyboarding. With the help of their [website](https://osb.moe/), this server aims to provide users with multiple learning resources and a regular showcase of community-made storyboards. |
+
+## Miscellaneous
+
+These are servers that do not fit in any specific category. While they may not have a defined category, they still serve a purpose in the community. Either by maintaining community tools, serving multiple purposes at the same time, or simply not focusing on any of the above categories.
```

Comment:
description for "Miscellaneous" is unnecessary imo

Reply:
I'll keep the first sentence maybe, just to be brief. Or would that look weird under a header?

Reply:
I think all of this is basically just a longer way to say what "miscellaneous" means. the heading is descriptive enough, imo. If you really want to keep it though, first sentence alone is ok

Reply:
Trying that then. If in the preview it looks weird, we can easily remove it then.

Reply:
(Weird, removed)